Client/Application/AppDelegate.swift

```diff
@@ -294,6 +368,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
                 url = item.value
             case "private":
                 isPrivate = NSString(string: item.value ?? "false").boolValue
+            case "deep-link":
+                guard let value = item.value else { break }
+                Router.shared.routeURL(value)
```

Comment:
The asserts are enabled for development/debug builds so in those cases we would hard crash (which is fine for testing) but will be silent on release builds. 

I'm pretty confident that the delegate method is always called on the main thread but if you're looking to guarantee it we can wrap this with something like:

```swift
if Thread.isMainThread {
    ...
} else {
    DispatchQueue.main.async {
        ...
    }
}
```